### DISCUSSION TITLE
Reload Gravity Forms on page change

### DISCUSSION THREAD

**Discussion by gonzalo-terrahq** [swup/swup#899] [Help]
### Describe your problem 🧐

Hello, I am using the Gravity Forms plugin and the Swup library for transitions between pages. Everything works correctly for me on the first load, but when changing I receive an error in the console `(cannont read Properties of undefined reading 'defaultPrevented')`.
I would like to know if there is any way to reinitialize Gravity Forms and its scripts to apply it to the Swup ContentReplaced hook. In the case of Contact Form 7 I have used `wpcf7.init()`, but in this case I cannot find anything similar.

Thank you.

### What is your question? 🙋‍♀️

How can I ensure that Gravity Forms runs after changing pages?

### URL to minimal reproduction 🔗

https://codepen.io/Gonzalo-Alcina/pen/YzbPbdO

### Checked all these? 📚

- [X] I have read and searched the [official docs](https://swup.js.org)
- [X] I have checked [discussions](https://github.com/orgs/swup/discussions) and [existing issues](https://github.com/issues?q=is:issue+org:swup) for answers to my question
- [X] I agree to follow this project's [Code of Conduct](https://github.com/swup/.github/blob/master/CODE_OF_CONDUCT.md)
- [X] I have provided all necessary information to the best of my knowledge

**Comment by hirasso** [comment]
You should post your question on the Support Forums of Gravity Forms, as it‘s not directly related to swup itself (only due to the fact that swup converts your website into a single page app).

we would be grateful if you could report back on the solution, should you find one.

**Comment by elliottmangham** [comment]
@gonzalo-terrahq did you ever find a solution to this?
